<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAddRadioInternet</class>
 <widget class="QDialog" name="FrmAddRadioInternet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radio en Internet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 48, 58);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>53</x>
     <y>100</y>
     <width>301</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="BtnPlay">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Cue</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/player/play.png</normaloff>:/general/iconos/player/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnStop">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Stop</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {

  background-color: rgb(32, 39, 47);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }


</string>
        </property>
        <property name="icon">
         <iconset resource="radit.qrc">
          <normaloff>:/general/iconos/player/stop.png</normaloff>:/general/iconos/player/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="BtnAceptar">
      <property name="styleSheet">
       <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BtnCancelar">
      <property name="styleSheet">
       <string notr="true">QPushButton {

background-color: rgb(207, 207, 207);
color: rgb(0, 0, 0);

    
 }

 

QPushButton:hover {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f5b14e, stop: 1 #fff9ef);
 }

</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/ficheros/iconos/ficheros/online.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>74</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);

</string>
        </property>
        <property name="text">
         <string>Nombre de la estación</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditNombre">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);

</string>
        </property>
        <property name="text">
         <string>Introduzca la duración de la conexión:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="TimeDuracion">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);

</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);

</string>
        </property>
        <property name="text">
         <string>Inserte la URL:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EditUrl">
        <property name="styleSheet">
         <string notr="true">border: 1px solid rgb(27, 128, 217);
color: rgb(255, 255, 255);
</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="radit.qrc"/>
 </resources>
 <connections/>
</ui>
